src/jit/codegenlegacy.cpp

```diff
@@ -18830,15 +18830,17 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
                                                    (ssize_t)stubAddr);
                             // The stub will write-back to this register, so don't track it
                             regTracker.rsTrackRegTrash(compiler->virtualStubParamInfo->GetReg());
-                            getEmitter()->emitIns_R_R_I(INS_ldr, EA_PTRSIZE, REG_JUMP_THUNK_PARAM,
```

Comment:
> R12 for this because the VM stubs required it.

https://github.com/dotnet/coreclr/blob/master/Documentation/botr/clr-abi.md only mentions the indirection cell address in r4. It does not say anything about r12, and I was not fine any stub that needs it.